server/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -216,7 +217,8 @@ public DocTableInfo(RelationName ident,
                         Version versionCreated,
                         @Nullable Version versionUpgraded,
                         boolean closed,
-                        Set<Operation> supportedOperations) {
+                        Set<Operation> supportedOperations,
+                        NodeContext nodeContext) {
```

Comment:
Can you revert this part?

The `NodeContext` is not a part of a table and doesn't belong here.
 
Either extend the `AlterTableOperator` interface with another parameter (and it doesn't need to extend a Function interface if that get's in the way, we don't make use of that anyway)

Or remove the static constant and pass it along when needed.